metadata-io/src/test/java/com/linkedin/metadata/search/elasticsearch/indexbuilder/MappingsBuilderTest.java

```diff
@@ -85,6 +85,11 @@ public void testMappingsBuilder() {
     assertTrue(nestedForeignKeySubfields.containsKey("keyword"));
     assertTrue(nestedForeignKeySubfields.containsKey("ngram"));
 
+    // OBJECT
+    Map<String, Object> esObjectField = (Map<String, Object>) properties.get("esObjectField");
```

Comment:
Overall my major comment is that we should rename esObjectField to something else.

Can we just call it "objectField" _everywhere_?

The idea of field types is to add some abstraction between us and elastic / the underlying search service. By putting a bunch of "esX" variables in the code it kind of moves us away from that goal!

Reply:
I can rename it. My thinking was that it's a test model so I named the field after the feature I wanted to test.